src/storage/v2/replication/enums.hpp

```diff
@@ -13,11 +13,7 @@
 #include <cstdint>
 
 namespace memgraph::storage::replication {
-enum class ReplicationRole : uint8_t { MAIN, REPLICA };
-
-enum class ReplicationMode : std::uint8_t { SYNC, ASYNC };
 
 enum class ReplicaState : std::uint8_t { READY, REPLICATING, RECOVERY, INVALID };
```

Comment:
I think this makes sense to decouple in a separate file.

Reply:
In another refactor